game.py

```diff
@@ -0,0 +1,62 @@
+import random
+
+
+class Card:
+    def __init__(self, value, suit):
+        self.value = value
+        self.suit = suit
+
+    def __str__(self):
+        return f"{self.value}{self.suit}"
+
+    def __repr__(self):
+        return f"Card('{self.value}', '{self.suit}')"
+
+
+class Deck:
+
+    CARD_VALUE = ["2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K", "A"]
+    CARD_SUIT = ["\u2663", "\u2665", "\u2666", "\u2660"]
```

Comment:
ChatGPT suggested:

```
self.cards = [Card(suit, rank) for suit in ['Hearts', 'Diamonds', 'Clubs', 'Spades'] for rank in range(1, 14)]
```

Reply:
How would you handle Ace?

Rules:
> Card Values: In blackjack, numbered cards from 2 to 10 are worth their face value. Face cards (Jack, Queen, and King) are each worth 10 points. An Ace can be worth either 1 or 11 points, depending on which value is more advantageous to the player.

Reply:
Checked Ben's repo quickly and he has this:

```
    def card_value(self):
        if self.rank in "J Q K".split():
            return 10
        elif self.rank == "A":
            return (1, 11)
        else:
            return int(self.rank)
```

And then he has some magic around setting ace, because it can be 1 or 11 "depending on which value is more advantageous" - maybe we table this in an issue?